--- src/WebKit-r172617/Source/PlatformGTK.cmake~	2014-08-15 02:13:39.000000000 +0900
+++ src/WebKit-r172617/Source/PlatformGTK.cmake	2014-08-26 12:08:13.933031356 +0900
@@ -1,7 +1,7 @@
 add_subdirectory(${WEBCORE_DIR}/platform/gtk/po)
 
 # This allows exposing a 'gir' target which builds all GObject introspection files.
-add_custom_target(gir ALL DEPENDS ${GObjectIntrospectionTargets})
+#add_custom_target(gir ALL DEPENDS ${GObjectIntrospectionTargets})
 
 if (ENABLE_WEBKIT2)
     list(APPEND DocumentationDependencies
--- src/WebKit-r172617/Source/JavaScriptCore/PlatformGTK.cmake~	2014-08-08 15:22:28.000000000 +0900
+++ src/WebKit-r172617/Source/JavaScriptCore/PlatformGTK.cmake	2014-08-26 12:25:00.822082577 +0900
@@ -1,13 +1,13 @@
 configure_file(javascriptcoregtk.pc.in ${CMAKE_BINARY_DIR}/Source/JavaScriptCore/javascriptcoregtk-${WEBKITGTK_API_VERSION}.pc @ONLY)
-configure_file(JavaScriptCore.gir.in ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir @ONLY)
+#configure_file(JavaScriptCore.gir.in ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir @ONLY)
 
-add_custom_command(
-    OUTPUT ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.typelib
-    DEPENDS ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
-    COMMAND ${INTROSPECTION_COMPILER} ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir -o ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.typelib
-)
+#add_custom_command(
+#    OUTPUT ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.typelib
+#    DEPENDS ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
+#    COMMAND ${INTROSPECTION_COMPILER} ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir -o ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.typelib
+#)
 
-ADD_TYPELIB(${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.typelib)
+#ADD_TYPELIB(${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.typelib)
 
 install(FILES "${CMAKE_BINARY_DIR}/Source/JavaScriptCore/javascriptcoregtk-${WEBKITGTK_API_VERSION}.pc"
         DESTINATION "${LIB_INSTALL_DIR}/pkgconfig"
@@ -23,12 +23,12 @@
         DESTINATION "${WEBKITGTK_HEADER_INSTALL_DIR}/JavaScriptCore"
 )
 
-install(FILES ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
-        DESTINATION ${INTROSPECTION_INSTALL_GIRDIR}
-)
-install(FILES ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.typelib
-        DESTINATION ${INTROSPECTION_INSTALL_TYPELIBDIR}
-)
+#install(FILES ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
+#        DESTINATION ${INTROSPECTION_INSTALL_GIRDIR}
+#)
+#install(FILES ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.typelib
+#        DESTINATION ${INTROSPECTION_INSTALL_TYPELIBDIR}
+#)
 add_definitions(-DSTATICALLY_LINKED_WITH_WTF)
 
 list(APPEND JavaScriptCore_LIBRARIES
--- src/WebKit-r172617/Source/WebKit2/PlatformGTK.cmake~	2014-08-08 15:22:29.000000000 +0900
+++ src/WebKit-r172617/Source/WebKit2/PlatformGTK.cmake	2014-08-26 12:45:21.205144658 +0900
@@ -726,102 +726,102 @@
 add_dependencies(webkit2gtkinjectedbundle GObjectDOMBindings)
 add_webkit2_prefix_header(webkit2gtkinjectedbundle)
 
-add_custom_command(
-    OUTPUT ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
-    DEPENDS WebKit2
-    DEPENDS ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
-    COMMAND CC=${CMAKE_C_COMPILER} CFLAGS=-Wno-deprecated-declarations LDFLAGS=
-        ${INTROSPECTION_SCANNER}
-        --quiet
-        --warn-all
-        --symbol-prefix=webkit
-        --identifier-prefix=WebKit
-        --namespace=WebKit2
-        --nsversion=${WEBKITGTK_API_VERSION}
-        --include=GObject-2.0
-        --include=Gtk-3.0
-        --include=Soup-2.4
-        --include-uninstalled=${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
-        --library=webkit2gtk-${WEBKITGTK_API_VERSION}
-        --library=javascriptcoregtk-${WEBKITGTK_API_VERSION}
-        -L${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
-        --no-libtool
-        --pkg=gobject-2.0
-        --pkg=gtk+-3.0
-        --pkg=libsoup-2.4
-        --pkg-export=webkit2gtk-${WEBKITGTK_API_VERSION}
-        --output=${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
-        --c-include="webkit2/webkit2.h"
-        -DBUILDING_WEBKIT
-        -DWEBKIT2_COMPILATION
-        -I${CMAKE_SOURCE_DIR}/Source
-        -I${WEBKIT2_DIR}
-        -I${JAVASCRIPTCORE_DIR}/ForwardingHeaders
-        -I${DERIVED_SOURCES_DIR}
-        -I${DERIVED_SOURCES_WEBKIT2GTK_DIR}
-        -I${FORWARDING_HEADERS_WEBKIT2GTK_DIR}
-        ${WebKit2GTK_INSTALLED_HEADERS}
-        ${WEBKIT2_DIR}/UIProcess/API/gtk/*.cpp
-)
+#add_custom_command(
+#    OUTPUT ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
+#    DEPENDS WebKit2
+#    DEPENDS ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
+#    COMMAND CC=${CMAKE_C_COMPILER} CFLAGS=-Wno-deprecated-declarations LDFLAGS=
+#        ${INTROSPECTION_SCANNER}
+#        --quiet
+#        --warn-all
+#        --symbol-prefix=webkit
+#        --identifier-prefix=WebKit
+#        --namespace=WebKit2
+#        --nsversion=${WEBKITGTK_API_VERSION}
+#        --include=GObject-2.0
+#        --include=Gtk-3.0
+#        --include=Soup-2.4
+#       --include-uninstalled=${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
+#       --library=webkit2gtk-${WEBKITGTK_API_VERSION}
+#        --library=javascriptcoregtk-${WEBKITGTK_API_VERSION}
+#        -L${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+#        --no-libtool
+#        --pkg=gobject-2.0
+#        --pkg=gtk+-3.0
+#        --pkg=libsoup-2.4
+#        --pkg-export=webkit2gtk-${WEBKITGTK_API_VERSION}
+#        --output=${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
+#       --c-include="webkit2/webkit2.h"
+#        -DBUILDING_WEBKIT
+#        -DWEBKIT2_COMPILATION
+#        -I${CMAKE_SOURCE_DIR}/Source
+#       -I${WEBKIT2_DIR}
+#        -I${JAVASCRIPTCORE_DIR}/ForwardingHeaders
+#        -I${DERIVED_SOURCES_DIR}
+#        -I${DERIVED_SOURCES_WEBKIT2GTK_DIR}
+#       -I${FORWARDING_HEADERS_WEBKIT2GTK_DIR}
+#        ${WebKit2GTK_INSTALLED_HEADERS}
+#        ${WEBKIT2_DIR}/UIProcess/API/gtk/*.cpp
+#)
+
+#add_custom_command(
+#    OUTPUT ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir
+#    DEPENDS ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
+#    DEPENDS ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
+#    COMMAND CC=${CMAKE_C_COMPILER} CFLAGS=-Wno-deprecated-declarations LDFLAGS=
+#        ${INTROSPECTION_SCANNER}
+#        --quiet
+#        --warn-all
+#        --symbol-prefix=webkit
+#        --identifier-prefix=WebKit
+#        --namespace=WebKit2WebExtension
+#        --nsversion=${WEBKITGTK_API_VERSION}
+#        --include=GObject-2.0
+#        --include=Gtk-3.0
+#        --include=Soup-2.4
+#        --include-uninstalled=${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
+#        --library=webkit2gtk-${WEBKITGTK_API_VERSION}
+#        --library=javascriptcoregtk-${WEBKITGTK_API_VERSION}
+#        -L${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+#        --no-libtool
+#        --pkg=gobject-2.0
+#        --pkg=gtk+-3.0
+#        --pkg=libsoup-2.4
+#        --pkg-export=webkit2gtk-web-extension-${WEBKITGTK_API_VERSION}
+#        --output=${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir
+#        --c-include="webkit2/webkit-web-extension.h"
+#        -DBUILDING_WEBKIT
+#        -DWEBKIT2_COMPILATION
+#        -I${CMAKE_SOURCE_DIR}/Source
+#        -I${WEBKIT2_DIR}
+#        -I${JAVASCRIPTCORE_DIR}/ForwardingHeaders
+#        -I${DERIVED_SOURCES_DIR}
+#        -I${DERIVED_SOURCES_WEBKIT2GTK_DIR}
+#        -I${FORWARDING_HEADERS_DIR}
+#        -I${FORWARDING_HEADERS_WEBKIT2GTK_DIR}
+#        -I${FORWARDING_HEADERS_WEBKIT2GTK_EXTENSION_DIR}
+#        -I${WEBKIT2_DIR}/WebProcess/InjectedBundle/API/gtk
+#        ${GObjectDOMBindings_GIR_HEADERS}
+#        ${WebKit2WebExtension_INSTALLED_HEADERS}
+#        ${WEBKIT2_DIR}/UIProcess/API/gtk/WebKitURIRequest.h
+#        ${WEBKIT2_DIR}/UIProcess/API/gtk/WebKitURIResponse.h
+#        ${WEBKIT2_DIR}/WebProcess/InjectedBundle/API/gtk/*.cpp
+#)
+
+#add_custom_command(
+#    OUTPUT ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.typelib
+#    DEPENDS ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
+#    COMMAND ${INTROSPECTION_COMPILER} --includedir=${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir -o ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.typelib
+#)
+
+#add_custom_command(
+#    OUTPUT ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.typelib
+#    DEPENDS ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir
+#    COMMAND ${INTROSPECTION_COMPILER} --includedir=${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir -o ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.typelib
+#)
 
-add_custom_command(
-    OUTPUT ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir
-    DEPENDS ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
-    DEPENDS ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
-    COMMAND CC=${CMAKE_C_COMPILER} CFLAGS=-Wno-deprecated-declarations LDFLAGS=
-        ${INTROSPECTION_SCANNER}
-        --quiet
-        --warn-all
-        --symbol-prefix=webkit
-        --identifier-prefix=WebKit
-        --namespace=WebKit2WebExtension
-        --nsversion=${WEBKITGTK_API_VERSION}
-        --include=GObject-2.0
-        --include=Gtk-3.0
-        --include=Soup-2.4
-        --include-uninstalled=${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
-        --library=webkit2gtk-${WEBKITGTK_API_VERSION}
-        --library=javascriptcoregtk-${WEBKITGTK_API_VERSION}
-        -L${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
-        --no-libtool
-        --pkg=gobject-2.0
-        --pkg=gtk+-3.0
-        --pkg=libsoup-2.4
-        --pkg-export=webkit2gtk-web-extension-${WEBKITGTK_API_VERSION}
-        --output=${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir
-        --c-include="webkit2/webkit-web-extension.h"
-        -DBUILDING_WEBKIT
-        -DWEBKIT2_COMPILATION
-        -I${CMAKE_SOURCE_DIR}/Source
-        -I${WEBKIT2_DIR}
-        -I${JAVASCRIPTCORE_DIR}/ForwardingHeaders
-        -I${DERIVED_SOURCES_DIR}
-        -I${DERIVED_SOURCES_WEBKIT2GTK_DIR}
-        -I${FORWARDING_HEADERS_DIR}
-        -I${FORWARDING_HEADERS_WEBKIT2GTK_DIR}
-        -I${FORWARDING_HEADERS_WEBKIT2GTK_EXTENSION_DIR}
-        -I${WEBKIT2_DIR}/WebProcess/InjectedBundle/API/gtk
-        ${GObjectDOMBindings_GIR_HEADERS}
-        ${WebKit2WebExtension_INSTALLED_HEADERS}
-        ${WEBKIT2_DIR}/UIProcess/API/gtk/WebKitURIRequest.h
-        ${WEBKIT2_DIR}/UIProcess/API/gtk/WebKitURIResponse.h
-        ${WEBKIT2_DIR}/WebProcess/InjectedBundle/API/gtk/*.cpp
-)
-
-add_custom_command(
-    OUTPUT ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.typelib
-    DEPENDS ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
-    COMMAND ${INTROSPECTION_COMPILER} --includedir=${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir -o ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.typelib
-)
-
-add_custom_command(
-    OUTPUT ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.typelib
-    DEPENDS ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir
-    COMMAND ${INTROSPECTION_COMPILER} --includedir=${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir -o ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.typelib
-)
-
-ADD_TYPELIB(${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.typelib)
-ADD_TYPELIB(${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.typelib)
+#ADD_TYPELIB(${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.typelib)
+#ADD_TYPELIB(${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.typelib)
 
 install(TARGETS webkit2gtkinjectedbundle
         DESTINATION "${LIB_INSTALL_DIR}/webkit2gtk-${WEBKITGTK_API_VERSION}/injected-bundle"
@@ -834,14 +834,14 @@
               ${WebKit2WebExtension_INSTALLED_HEADERS}
         DESTINATION "${WEBKITGTK_HEADER_INSTALL_DIR}/webkit2"
 )
-install(FILES ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
-              ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir
-        DESTINATION ${INTROSPECTION_INSTALL_GIRDIR}
-)
-install(FILES ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.typelib
-              ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.typelib
-        DESTINATION ${INTROSPECTION_INSTALL_TYPELIBDIR}
-)
+#install(FILES ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.gir
+#              ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir
+#        DESTINATION ${INTROSPECTION_INSTALL_GIRDIR}
+#)
+#install(FILES ${CMAKE_BINARY_DIR}/WebKit2-${WEBKITGTK_API_VERSION}.typelib
+#              ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.typelib
+#        DESTINATION ${INTROSPECTION_INSTALL_TYPELIBDIR}
+#)
 
 file(WRITE ${CMAKE_BINARY_DIR}/gtkdoc-webkit2gtk.cfg
     "[webkit2gtk]\n"
