--- src/glibc-2.19/sysdeps/generic/ldsodefs.h~	2014-02-07 18:04:38.000000000 +0900
+++ src/glibc-2.19/sysdeps/generic/ldsodefs.h	2014-08-28 14:32:24.603957677 +0900
@@ -392,7 +392,7 @@
 #define TLS_SLOTINFO_SURPLUS (62)
 
 /* Number of additional slots in the dtv allocated.  */
-#define DTV_SURPLUS	(14)
+#define DTV_SURPLUS	(28)
 
   /* Initial dtv of the main thread, not allocated with normal malloc.  */
   EXTERN void *_dl_initial_dtv;
