--- src/wxWidgets-3.0.1/configure~	2014-06-15 06:48:48.000000000 +0900
+++ src/wxWidgets-3.0.1/configure	2014-08-15 00:14:37.126192051 +0900
@@ -23340,7 +23340,7 @@
     no_gtk=yes
   fi
 
-  min_gtk_version=2.6.0
+  min_gtk_version=2.24.24
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for GTK+ - version >= $min_gtk_version" >&5
 $as_echo_n "checking for GTK+ - version >= $min_gtk_version... " >&6; }
 
