--- src/wxWidgets-3.0.1/configure~	2014-06-15 06:48:48.000000000 +0900
+++ src/wxWidgets-3.0.1/configure	2014-08-24 08:38:34.014346527 +0900
@@ -23604,7 +23604,7 @@
     no_gtk=yes
   fi
 
-  min_gtk_version=3.0.0
+  min_gtk_version=3.13.6
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for GTK+ - version >= $min_gtk_version" >&5
 $as_echo_n "checking for GTK+ - version >= $min_gtk_version... " >&6; }
 
