--- src/sdl2/Makefile.in~	2014-08-27 18:07:57.770220406 +0900
+++ src/sdl2/Makefile.in	2014-08-27 18:39:49.851317674 +0900
@@ -151,17 +151,17 @@
 install: all install-bin install-hdrs install-lib install-data
 install-bin:
 	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(bindir)
-	$(INSTALL) -m 755 sdl2-config $(DESTDIR)$(bindir)/sdl2-config
+	$(INSTALL) -m 755 sdl2-config $(DESTDIR)$(bindir)/sdl2-config($EXEXT)
 install-hdrs: update-revision
-	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(includedir)/SDL2
+	$(SHELL) $(auxdir)/mkinstalldirs $(DESTDIR)$(libdir)/sdl2/include
 	for file in $(HDRS) $(SDLTEST_HDRS); do \
-	    $(INSTALL) -m 644 $(srcdir)/include/$$file $(DESTDIR)$(includedir)/SDL2/$$file; \
+	    $(INSTALL) -m 644 $(srcdir)/include/$$file $(DESTDIR)$(libdir)/sdl2/include/$$file; \
 	done
-	$(INSTALL) -m 644 include/SDL_config.h $(DESTDIR)$(includedir)/SDL2/SDL_config.h
+	$(INSTALL) -m 644 include/SDL_config.h $(DESTDIR)$(libdir)/sdl2/include/SDL_config.h
 	if test -f include/SDL_revision.h; then \
-	    $(INSTALL) -m 644 include/SDL_revision.h $(DESTDIR)$(includedir)/SDL2/SDL_revision.h; \
+	    $(INSTALL) -m 644 include/SDL_revision.h $(DESTDIR)$(libdir)/sdl2/include/SDL_revision.h; \
 	else \
-	    $(INSTALL) -m 644 $(srcdir)/include/SDL_revision.h $(DESTDIR)$(includedir)/SDL2/SDL_revision.h; \
+	    $(INSTALL) -m 644 $(srcdir)/include/SDL_revision.h $(DESTDIR)$(libdir)/sdl2/include/SDL_revision.h; \
 	fi
 
 install-lib: $(objects) $(objects)/$(TARGET) $(objects)/$(SDLMAIN_TARGET) $(objects)/$(SDLTEST_TARGET)
@@ -182,11 +182,11 @@
 	rm -f $(DESTDIR)$(bindir)/sdl2-config
 uninstall-hdrs:
 	for file in $(HDRS) $(SDLTEST_HDRS); do \
-	    rm -f $(DESTDIR)$(includedir)/SDL2/$$file; \
+	    rm -f $(DESTDIR)$(libdir)/sdl2/include/SDL2/$$file; \
 	done
-	rm -f $(DESTDIR)$(includedir)/SDL2/SDL_config.h
-	rm -f $(DESTDIR)$(includedir)/SDL2/SDL_revision.h
-	-rmdir $(DESTDIR)$(includedir)/SDL2
+	rm -f $(DESTDIR)$(libdir)/sdl2/include/SDL_config.h
+	rm -f $(DESTDIR)$(libdir)/sdl2/include/SDL_revision.h
+	-rmdir $(DESTDIR)$(libdir)/sdl2/include/SDL2
 uninstall-lib:
 	$(LIBTOOL) --mode=uninstall rm -f $(DESTDIR)$(libdir)/$(TARGET)
 	rm -f $(DESTDIR)$(libdir)/$(SDLMAIN_TARGET)
--- src/sdl2/sdl2.pc.in~	2014-08-27 18:07:57.804220408 +0900
+++ src/sdl2/sdl2.pc.in	2014-08-27 18:41:06.146321555 +0900
@@ -3,7 +3,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
-includedir=@includedir@
+includedir=@libdir@/sdl2/include/
 
 Name: sdl2
 Description: Simple DirectMedia Layer is a cross-platform multimedia library designed to provide low level access to audio, keyboard, mouse, joystick, 3D hardware via OpenGL, and 2D video framebuffer.
@@ -12,4 +12,4 @@
 Conflicts:
 Libs: -L${libdir} @SDL_RLD_FLAGS@ @SDL_LIBS@
 Libs.private: @SDL_STATIC_LIBS@
-Cflags: -I${includedir}/SDL2 @SDL_CFLAGS@
+Cflags: -I${includedir} @SDL_CFLAGS@
--- src/sdl2/sdl2-config.in~	2014-08-27 18:07:57.803220408 +0900
+++ src/sdl2/sdl2-config.in	2014-08-27 18:43:05.609327632 +0900
@@ -42,7 +42,7 @@
       echo @SDL_VERSION@
       ;;
     --cflags)
-      echo -I@includedir@/SDL2 @SDL_CFLAGS@
+      echo -I@libdir@/sdl2/include @SDL_CFLAGS@
       ;;
 @ENABLE_SHARED_TRUE@    --libs)
 @ENABLE_SHARED_TRUE@      echo -L@libdir@ @SDL_RLD_FLAGS@ @SDL_LIBS@
#--- src/sdl2/CMakeLists.txt~	2014-08-27 18:48:45.991344947 +0900
#+++ src/sdl2/CMakeLists.txt	2014-08-27 19:01:32.687383949 +0900
#@@ -1316,7 +1316,7 @@
#   list(REMOVE_ITEM INCLUDE_FILES ${SDL2_SOURCE_DIR}/include/${_INCNAME})
# endforeach()
# list(APPEND INCLUDE_FILES ${BIN_INCLUDE_FILES})
#-install(FILES ${INCLUDE_FILES} DESTINATION include/SDL2)
#+install(FILES ${INCLUDE_FILES} DESTINATION lib32/sdl2/include)
# 
# if(NOT WINDOWS OR CYGWIN)
#   if(SDL_SHARED)
